sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -6,6 +6,7 @@
 import os
 import ast
 
+from sonic_platform.device_data import DeviceDataManager
```

Comment:
Move to inside the mellanox if - It won't work on other platforms

Reply:
Great point, thanks.
Done.